Emby.Dlna/Profiles/Xml/Default.xml

```diff
@@ -2,11 +2,11 @@
 <Profile xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
   <Name>Generic Device</Name>
   <Manufacturer>Emby</Manufacturer>
-  <ManufacturerUrl>https://emby.media</ManufacturerUrl>
+  <ManufacturerUrl>https://github.com/jellyfin/jellyfin</ManufacturerUrl>
```

Comment:
I thought we had a site named "jellyfin.media", maybe change to it instead of direct link to repo?

Reply:
Agreed here, let's use `jellyfin.media` wherever possible.